### ISSUE TITLE
`^` undefined when using function as the `chebop` operator

### ISSUE THREAD

**Issue by MarcinKonowalczyk** [chebfun/chebfun#2287]
Hello,

I came across a problem when trying to solve IVP with a backslash command and a `chebop`.
It works perfectly well when I do this:

    close all; clear; clc;
    p1 = 1 + 0i; p2 = -1 + 0i; % Two fixed unit masses
    N = chebop(0,10);
    N.op = @(t,Y) diff(Y,2) + (Y-p1)/abs((Y-p1))^3 + (Y-p2)/abs((Y-p2))^3;
    N.lbc = [0+0i 0.8+0.8i]; % Inital position and velocity
    u = N\0; % Solve for time evolution of position

    figure; axes; hold on;
    plot(u);
    plot(real(p1),imag(p1),'kx'); plot(real(p2),imag(p2),'kx');
    xlim([-2 2]); ylim([-2 2]); grid on; box on;

but throws an error when I replace the `N.op = ...` line with 
    
    N.op = @(t,Y) my_little_odefun(t,Y,p1,p2);

where `my_little_odefun.m` is simply

    function out = my_little_odefun(t,Y,p1,p2)
    out = diff(Y,2) + (Y-p1)/abs((Y-p1))^3 + (Y-p2)/abs((Y-p2))^3;
    end

This is the error stack I get:

    Undefined operator '^' for input arguments of type 'treeVar'.
    Error in my_little_odefun (line 2)
    out = diff(Y,2) + (Y-p1)/abs((Y-p1))^3 + (Y-p2)/abs((Y-p2))^3;
    Error in @(t,Y)my_little_odefun(t,Y,p1,p2)
    
    Error in treeVar.toFirstOrder (line 75)
            fevalResult = funIn(t, args{:});
    Error in chebop/solveivp (line 142)
            treeVar.toFirstOrder(N.op, rhs, N.domain, nVars, cellArg);
    Error in  \  (line 49)
        [varargout{1:nargout}] = solveivp(N, rhs, pref, varargin{:});

In other words the anonymous function gets parsed differently to a normal one.

I have Matlab 2017a running on MacOS Mojave 10.14.2. I've checked out `chebfun` fresh just today.

**Comment by nickhale** [comment]
Hi @MarcinKonowalczyk 

It seems Chebop isn't vectorizing the operator when it's defined via a function. (For those in the know, I believe this is because `mpower` and `mrdivide` haven't been implemented for the `@treeVar' class.)

@MarcinKonowalczyk: You can make it work by simply using the following:
```
function out = my_little_odefun(t,Y,p1,p2)
out = diff(Y,2) + (Y-p1)./abs((Y-p1)).^3 + (Y-p2)./abs((Y-p2)).^3;
end
```
